### DISCUSSION TITLE
Reporting status bug

### DISCUSSION THREAD

**Discussion by tt053626** [open-sdg/open-sdg#1543] [Q&A] [answered]
Our goal 1 and goal 3 of the SDG website have wrong reporting status, but the rests are all correct.
The first goal has in total 3 indicators but the status shows only 2.
The third goal has 5, but shows in reporting status 6.

Here is the website link: https://beyersar.github.io/SDGTK/reporting-status/
Site repo: https://github.com/beyersar/SDGTK
Data repo: https://github.com/beyersar/DataTK

**Comment by LucyGwilliamAdmin** [comment, marked as answer]
@tt053626 if you look at your [metadata for 1.3.1](https://github.com/beyersar/DataTK/blob/develop/meta/1-3-1.md), the field `sdg_goal` is set to 3 instead of 1

Also omitting the field completely will have the same affect as that field isn't actually required anymore, unless you are planning on showing it on the platform somewhere.

I can see the reporting values for goal 3 are actually different now so are they correct?

**Reply by tt053626** [reply]
@LucyGwilliamAdmin You're right! It is the metadata that was wrong. We updated the data for goal 3 so it looks a bit different, but the problem was the same. Thank you so much!
